Document how to add schema vocabulary definitions to ckanext-datagovuk

AlanGabbianelli opened the pull request:
Trello card: https://trello.com/c/KkaFHYpE/749-add-a-new-schema-to-datagovuk
Related PR: https://github.com/alphagov/ckanext-datagovuk/pull/139

brucebolt commented:
Just another thought: it could be useful to document where this appears in the user interface

AlanGabbianelli commented:
@brucebolt I updated the wording to `schema vocabulary definition` and moved it to a separate section of the docs.

I don't know where this appears in the user interface. Can you please advise on this?
